lib/core/src/swapper/boltz_status_stream.rs

```diff
@@ -140,7 +140,9 @@ impl SwapperStatusStream for BoltzStatusStream {
                                                 Err(e) => warn!("WS response is invalid SwapUpdate: {e:?}"),
                                             }
                                         },
-                                        Ok(msg) => info!("Unhandled msg: {msg:?}"),
+                                        Ok(Message::Ping(_)) => debug!("Received ping"),
```

Comment:
The nice thing to do here would be to send back a Pong

Reply:
Yep, tungstenite does that automatically, here it only surfaces the received ping event.

Reply:
Cool 👍